src/components/GridContent.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import img5 from '../img/img5.svg';
+import { Link } from 'react-router-dom';
+import { Grid, Paper, Typography, Button } from '@mui/material';
+
+function GridContent() {
```

Comment:
I wasn't sure what to call this.

Reply:
Hmmm, how about `LookingForScholarshipsBanner`?